example/gtk/gtkmm_main.cpp

```diff
@@ -1,5 +1,9 @@
 #include "latex.h"
+#ifdef WITH_MM
+#include "graphic_cairomm.h"
```

Comment:
should be removed since there are no `cairomm` impl

Reply:
fixed